ban/http/resources.py

```diff
@@ -68,6 +68,7 @@ def __new__(mcs, name, bases, attrs, **kwargs):
 class BaseCRUD(BaseCollection, metaclass=WithURL):
 
     order_by = None
+    param_whitelist = None
```

Comment:
Use `[]` as default, otherwise `get_where_clause` will crash for any other resource than `PostCode`.